applications/asset_tracker_v2/src/ext_sensors/ext_sensors.h

```diff
@@ -27,7 +27,8 @@ extern "C" {
 
 /** @brief Enum containing callback events from library. */
 enum ext_sensor_evt_type {
-	EXT_SENSOR_EVT_ACCELEROMETER_TRIGGER,
+	EXT_SENSOR_EVT_ACCELEROMETER_LP_TRIGGER,
```

Comment:
Could add some documentation for these that specifies writes out LP and HG and the equivalent sensor devices. ADXL362 and 372

Reply:
done